scripts/test-harness-contracts.ps1

```
@@ -170,6 +170,49 @@ try {
+    Check-Match '.claude/skills/pelizzai-finish/SKILL.md' ':\(exclude\)pelizzai/data/verification-standard\.md' 'finish: product guard excludes the ratified standard'
+    Check-Match '.claude/skills/pelizzai-execute/references/delivery-seal.md' 'Stale cursor[\s\S]{0,400}merge-base --is-ancestor' 'delivery-seal: stale cursor observed against git'
+    Check-Match '.claude/skills/pelizzai-execute/references/delivery-seal.md' '.pelizzai-finish. §1\.6 candidates read from the migrated block' 'delivery-seal: recovery presents the write-back candidates'
+    Check-Match '.claude/skills/pelizzai-execute/references/delivery-seal.md' 'ONE atomic movement' 'delivery-seal: the recovery is one atomic movement (review round 1)'
+    Check-Match '.claude/skills/pelizzai-execute/references/delivery-seal.md' 'SAME metadata\s+commit' 'delivery-seal: ratified knowledge rides the same metadata commit (review round 1)'
+    Check-Match '.claude/skills/pelizzai-execute/references/delivery-seal.md' 'full SHA \(a .<none>./missing value never enters this\s+path\)' 'delivery-seal: stale path requires a full validated-head (review round 1)'
+    Check-Match '.claude/skills/pelizzai-resume/SKILL.md' 'Stale cursor short of .delivered.' 'resume: stale pre-delivered cursor is not WIP divergence'
+    Check-Match '.claude/skills/pelizzai-resume/SKILL.md' '.validated-head. is a full SHA \(a .<none>./missing value never enters this path\)' 'resume: stale path requires a full validated-head (review rounds 1-2)'
+    Check-Match '.claude/skills/pelizzai-execute/references/delivery-seal.md' 'append the\s+same observation[\s\S]{0,80}pelizzai/data/history/<YYYY-MM-DD>-<slug>\.md' 'delivery-seal: the recovery stamps the history file too (review round 2)'
+    Check-Match '.claude/skills/pelizzai-finish/SKILL.md' 'Entry already present with .status: candidate. or .promoted.' 'finish: only candidate/promoted entries count as recurrence (review round 2)'
+    Check-Match '.claude/skills/pelizzai-finish/SKILL.md' 'Entry present but .retired.[^\n]*no count, no promotion' 'finish: a retired entry only flags the wrong retirement (review round 2)'
+    Check-Match '.claude/skills/pelizzai-finish/SKILL.md' 'symptom similarity is not identity' 'finish: recurrence identity is deterministic (review round 1)'
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Check-Match' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)